terraform/paas/data.tf

```diff
@@ -0,0 +1,53 @@
+data azurerm_key_vault vault {
+  name                = var.azure_key_vault
+  resource_group_name = var.azure_resource_group
+}
+
+data azurerm_key_vault_secret Application {
```

Comment:
Any reason why it's uppercase?

Reply:
Because I typed it in upper case probably, no technical reason, so I have lower cased it.